psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -822,89 +826,111 @@ SharedMatrix MintsHelper::ao_shell_getter(const std::string &label, std::shared_
+SharedMatrix MintsHelper::ao_f12_scaled(std::shared_ptr <CorrelationFactor> corr)
+{
+    std::shared_ptr <TwoBodyAOInt> ints(integral_->f12_scaled(corr));
     return ao_helper("AO F12 Scaled Tensor", ints);
 }
 
-SharedMatrix MintsHelper::ao_f12_scaled(std::shared_ptr<CorrelationFactor> corr, std::shared_ptr<BasisSet> bs1,
-                                        std::shared_ptr<BasisSet> bs2, std::shared_ptr<BasisSet> bs3,
-                                        std::shared_ptr<BasisSet> bs4) {
+SharedMatrix MintsHelper::ao_f12_scaled(std::shared_ptr <CorrelationFactor> corr,
+                                        std::shared_ptr <BasisSet> bs1,
+                                        std::shared_ptr <BasisSet> bs2,
+                                        std::shared_ptr <BasisSet> bs3,
+                                        std::shared_ptr <BasisSet> bs4)
+{
     IntegralFactory intf(bs1, bs2, bs3, bs4);
-    std::shared_ptr<TwoBodyAOInt> ints(intf.f12_scaled(corr));
+    std::shared_ptr <TwoBodyAOInt> ints(intf.f12_scaled(corr));
     return ao_helper("AO F12 Scaled Tensor", ints);
 }
 
-SharedMatrix MintsHelper::ao_f12_squared(std::shared_ptr<CorrelationFactor> corr) {
-    std::shared_ptr<TwoBodyAOInt> ints(integral_->f12_squared(corr));
+SharedMatrix MintsHelper::ao_f12_squared(std::shared_ptr <CorrelationFactor> corr)
+{
+    std::shared_ptr <TwoBodyAOInt> ints(integral_->f12_squared(corr));
     return ao_helper("AO F12 Squared Tensor", ints);
 }
 
-SharedMatrix MintsHelper::ao_f12_squared(std::shared_ptr<CorrelationFactor> corr, std::shared_ptr<BasisSet> bs1,
-                                         std::shared_ptr<BasisSet> bs2, std::shared_ptr<BasisSet> bs3,
-                                         std::shared_ptr<BasisSet> bs4) {
+SharedMatrix MintsHelper::ao_f12_squared(std::shared_ptr <CorrelationFactor> corr,
+                                         std::shared_ptr <BasisSet> bs1,
+                                         std::shared_ptr <BasisSet> bs2,
+                                         std::shared_ptr <BasisSet> bs3,
+                                         std::shared_ptr <BasisSet> bs4)
+{
     IntegralFactory intf(bs1, bs2, bs3, bs4);
-    std::shared_ptr<TwoBodyAOInt> ints(intf.f12_squared(corr));
+    std::shared_ptr <TwoBodyAOInt> ints(intf.f12_squared(corr));
     return ao_helper("AO F12 Squared Tensor", ints);
 }
 
-SharedMatrix MintsHelper::ao_3coverlap_helper(const std::string &label, std::shared_ptr<ThreeCenterOverlapInt> ints) {
-    std::shared_ptr<BasisSet> bs1 = ints->basis1();
-    std::shared_ptr<BasisSet> bs2 = ints->basis2();
-    std::shared_ptr<BasisSet> bs3 = ints->basis3();
+SharedMatrix MintsHelper::ao_3coverlap_helper(const std::string &label, std::shared_ptr<ThreeCenterOverlapInt> ints)
+{
+    std::shared_ptr <BasisSet> bs1 = ints->basis1();
+    std::shared_ptr <BasisSet> bs2 = ints->basis2();
+    std::shared_ptr <BasisSet> bs3 = ints->basis3();
```

Comment:
Yea, it looks like when you rebased you flipped a lot of the clang-format changes. Could you re-apply clang format to MintsHelper? @robertodr can probably help you locally.

Reply:
Sure. I will try clang formatting MintsHelper.